site/docs/aliyun.md

```diff
@@ -0,0 +1,145 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Iceberg Aliyun Integrations
+
+Iceberg provides integration with different Aliyun services through the `iceberg-aliyun` module.
+This section describes how to use Iceberg with [Aliyun](https://www.alibabacloud.com/).
+
+## Enabling Aliyun Integration
+
+The `iceberg-aliyun` module integrates alibaba cloud services (Aliyun OSS, Aliyun DLF etc) with apache iceberg.
+Currently, it provides the bundled `iceberg-aliyun-runtime` module for users to access the iceberg table backed in
+alibaba cloud services. To enable the aliyun integration, people only need to ensure that the `iceberg-aliyun-runtime`
+jar is loaded into classpath correctly by the engines such as Spark, Flink, Hive, Presto etc.
+
+## Catalogs
+
+[Aliyun DLF](https://www.aliyun.com/product/bigdata/dlf) is a core service from Alibaba Cloud that satisfies users'
+needs for data asset management while creating data lake tables. DLF provides unified metadata views and permission
+management for data available in OSS ([Aliyun Object Storage Service](https://www.alibabacloud.com/product/object-storage-service)).
+It also provides real-time lake migration and cleaning templates for data and production-level metadata services for
+upper-layer data analysis engines. Aliyun DLF is a good choice to manage the apache iceberg tables, the Aliyun DLF
+catalog integration will come in the next releases.
+
+### Engines Access.
+
+All the engines (Spark, Hive, Flink, Presto) can access the iceberg table backed in aliyun services. There are following
+examples to show how to access it.
+
+The ideal way to show the example is using Aliyun DLF Catalog to manage those iceberg tables, but we still don't finish
```

Comment:
This part is a bit hard to read. Maybe this Engine Access section can be condensed to something like:

> All engines in aliyun services can access iceberg tables. Although the Iceberg and DLF integration has not been released yet, here are some examples of how you can manage Iceberg tables in a Hive catalog.